index.html

```diff
@@ -2579,9 +2588,16 @@
             <code>String</code>, return <code>false</code> and abort these steps.
           </li>
           <li>
+            Let <code>subdomain_pattern</code> be the result of prepending <code>"."</code>
```

Comment:
I think that in spec sense you don't need to use _ - you can do normal language <code>sub domain pattern</code>

We do that in manifest etc